ssms/basic_simulators/drift_functions.py

```diff
@@ -132,3 +132,126 @@ def ds_conflict_drift(
     v_t = (w_t * coherence_t) + (w_d * coherence_d)
 
     return v_t  # , w_t, w_d
+
+def SSP_drift(
+    t=np.arange(0, 20, 0.001),
+    p=1,          
+    sda = 0.1,    
+    rd = 0.05,    
+    congruency=1   
+):
+    """This drift is inspired by shrinking spotlight model in conflict task which
```

Comment:
this one doesn't have citation? (I assume this is what you were out to test in your own work then?)